### ISSUE TITLE
Suggestion for /watched

### ISSUE THREAD

**Issue by fclaussen** [Galarzaa90/NabBot#31]
Now that I see how /watched works, it would be nice to have a subcommand for details.

I would add someone like this
/watched add Maxie "Stole my loot"

And I would run something like /watched details Maxie to see why she was added to the list.

Don't know.. it would be nice to have some sort of description.

**Comment by fclaussen** [comment]
Another suggestion is to order people for level or vocation.
I added 2 guilds to test it and I now have a huge list. 

Not sure if that is possible though

**Comment by Galarzaa90** [comment]
Yes, this is definitely something I wanted to add, but I probably should show that reason on the watched list itself, cause there's a 2000 character limit on the message, so I would have to show it somewhere else.

The state where the watched list feature is right now is pretty basic, it was just an initial release just to have that feature, but it definitely needs a lot of work. Like maybe instead of keeping a list, sending messages when someone logs in and logs out, or a mix of both.

The task that updates it also needs optimization

So yes, definitely something I need to do, but still needs a lot of details.

**Comment by fclaussen** [comment]
I'm so excited about this project I'm thinking about learning python. lol

**Comment by CuJo38** [comment]
To expand on it, Maybe should be its own heading, but ill add here and let you decide. The Stalk and watched commands could also potentially check the experience gained and lost via the High scores. It is only limited to characters in the top 300's. Maybe once per day, or i guess on command would work better, /exp could display all discord members who are able to be tracked in high scores, and /expall could display all characters, discord and watched/stalked, to see the exp gains and losses if they are in the high scores list

**Comment by Sinsae** [comment]
I've had a few request on the exp gain too. Also a command to show the Tibia server down time. I guess some feel it hints on possible raids. 

/downtime amera

NabBot: On the last server save Amera servers were off for 8 minutes.

**Comment by CuJo38** [comment]
Also, as for functionality in the discord im hosting, I have multiple guilds set off in sub channels, but a main tibia one for all people in my discord (where all level and deaths go). If there was ever a function to have different watched lists and be able to have multiple spots they show up, So i could have one 'watched' list serve as a "who is online" funtion rather than using the /online function) and Wouldn't find having a separate one or more for tracking specific guild enemies, or players of interest, and of course a msg next to the name with why on there. Right now I just have the few guilds show up in one spot so its easier to find hunting partners with friendly guilds who aren't necessarily in my disc.

**Comment by Sinsae** [comment]
@CuJo38 I edited my 'watched list' to hold every guild in Solidera, and have it setup where the list isn't reposted every check, but edited. 10x less notifications. It's close as I've gotten to a direct online list.
And I know from experience its a pain to re add new guilds, or remove non existant guilds, and having to re-add them all, so it stays in alphabetical order.
I will try to add the tracking.py in the form of a cog, of what I use for that, since there isnt a character max limit since the message is split into parts, saves the message ID's, then edits the previous messages on the next check. Renaming a couple things and loading it as a 'cog' might allow you to run multiple watched lists.

**Comment by Galarzaa90** [comment]
@CuJo38 I do want to add something similar to the Watched List but for /online, to show who's online from the current discord.

So exactly what /online does, but on a fixed channel, so you can always look at it instead of having to use the command.

Watched List was something that was done experimentally and I haven't been able to really define the correct implementation for it.

**Comment by Galarzaa90** [comment]
Finally working on this:
![image](https://user-images.githubusercontent.com/12865379/40336522-ef937588-5d1e-11e8-8dcf-cbebea094442.png)

**Comment by Galarzaa90** [comment]
Implemented in be887cbcca71f5eeaa32160a8dda7f3a6dbf35fa, will be available on **v1.1.0**
